app/models/user.rb

```diff
@@ -364,6 +364,14 @@ def set_current_taxonomies
     TopbarSweeper.expire_cache(self)
   end
 
+  def my_organizations
```

Comment:
Ad linking to other comments in the PR https://github.com/theforeman/foreman/pull/3732#discussion_r74947625